src/styles/_catalog.scss

```diff
@@ -1,6 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
```

Comment:
```suggestion
  justify-items: center;
```
because now it sticks to left side
<img width="850" alt="image" src="https://user-images.githubusercontent.com/53611910/201545791-7a561f53-8442-4ed6-82b9-c27df107e873.png">